pmm/v3/pmm3-image-scanning.groovy

```diff
@@ -21,11 +21,27 @@ pipeline {
             name: 'TAG')
     }
     stages {
-        stage('Scan docker image') {
+        stage('Install Snyk CLI') {
             steps {
-                sh "lw-scanner image evaluate ${IMAGE} ${TAG} --html --html-file report.html --data-directory ."
-                archiveArtifacts 'report.html'
-                archiveArtifacts 'evaluations/**/**/evaluation_*.json'
+                script {
+                    sh '''
+                        curl -sL https://static.snyk.io/cli/latest/snyk-linux -o snyk
+                        chmod +x snyk
+                        npm install snyk-to-html
```

Comment:
```suggestion
                        npm install -g snyk-to-html
```
This will make redundant adding the `node_modules/.bin` to the PATH variable (see above).

Reply:
```
12:11:11  + npm install -g snyk-to-html
12:11:26  npm error code EACCES
12:11:26  npm error syscall mkdir
12:11:26  npm error path /usr/lib/node_modules/snyk-to-html
12:11:26  npm error errno -13
12:11:26  npm error [Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/snyk-to-html'] {
12:11:26  npm error   errno: -13,
12:11:26  npm error   code: 'EACCES',
12:11:26  npm error   syscall: 'mkdir',
12:11:26  npm error   path: '/usr/lib/node_modules/snyk-to-html'
12:11:26  npm error }
12:11:26  npm error
12:11:26  npm error The operation was rejected by your operating system.
12:11:26  npm error It is likely you do not have the permissions to access this file as the current user
12:11:26  npm error
12:11:26  npm error If you believe this might be a permissions issue, please double-check the
12:11:26  npm error permissions of the file and its containing directories, or try running
12:11:26  npm error the command again as root/Administrator.
12:11:26  
12:11:26  npm error A complete log of this run can be found in: /home/ec2-user/.npm/_logs/2024-11-13T07_11_18_469Z-debug-0.log
```